### PR TITLE
Remove the "2 Reviews Needed" column automation from the PR board

### PR DESCRIPTION
**Pull request by zackkrida**
## Description

This PR removes JavaScript logic from the `automations/js/src/project_automation/prs.mjs` file which moves cards into the "Needs 2 Reviews" column of our [PR project board](https://github.com/orgs/WordPress/projects/98?query=sort%3Aupdated-desc+is%3Aopen). 

```diff
-  } else {
-   await prBoard.moveCard(prCard.id, prBoard.columns.Needs2Reviews)
-  }
```

This PR _also_ updates the entire file to conform to our prettier config.

## Testing Instructions

Review the logic and hope it works!

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: automations/js/src/project_automation/prs.mjs [hunk 1]
```diff
@@ -19,17 +19,15 @@ async function syncReviews(core, pr, prBoard, prCard) {
...
-    core.info('Moving PR on the basis of review decision.')
+  if (reviewDecision === "APPROVED") {
+    core.info("Moving PR on the basis of review decision.")
     await prBoard.moveCard(prCard.id, prBoard.columns.Approved)
-  } else if (reviewDecision === 'CHANGES_REQUESTED') {
-    core.info('Moving PR on the basis of review decision.')
+  } else if (reviewDecision === "CHANGES_REQUESTED") {
+    core.info("Moving PR on the basis of review decision.")
     await prBoard.moveCard(prCard.id, prBoard.columns.ChangesRequested)
   } else if (reviewCounts.APPROVED === 1) {
```
Comment (dhruvkb) [inline automations/js/src/project_automation/prs.mjs:28]: This condition is unreachable, instead this should be the catch-all block.
```suggestion
  } else {
```

File: automations/js/src/project_automation/prs.mjs [hunk 2]
```diff
@@ -19,17 +19,15 @@ async function syncReviews(core, pr, prBoard, prCard) {
...
-    core.info('Moving PR on the basis of 1 approval.')
+    core.info("Moving PR on the basis of 1 approval.")
```
Comment (dhruvkb) [inline automations/js/src/project_automation/prs.mjs:29]: This line should be removed, it's the wrong message.
```suggestion
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]

**Comment by zackkrida** [comment]
@dhruvkb is the reasoning for your suggestions that having 1 approving review is the same as being "APPROVED", now? I just want to make sure I understand how this is working; thanks!

**Comment by dhruvkb** [comment]
Yeah, that's exactly it @zackkrida. PRs with one approval will already have `reviewDecision === "APPROVED"` and we need to send PRs to "Needs 1 Review" by default.

**Review by dhruvkb — APPROVED** [review]

**Review by AetherUnbound — APPROVED** [review]
LGTM!

**Review by obulat — APPROVED** [review]
The logic looks correct now.
We could rename the column to "Needs review" instead of "Needs 1 review" for brevity.

How did the file pass the linting step in CI before? Are these files not linted?

**Review by obulat — APPROVED** [review]
The logic looks correct now.
We could rename the column to "Needs review" instead of "Needs 1 review" for brevity.

~~How did the file pass the linting step in CI before? Are these files not linted?~~
The PR is failing the linting check. Do we use a different set of rules for these files?

**Comment by dhruvkb** [comment]
@obulat automations have [their separate Prettier config file](https://github.com/WordPress/openverse/blob/b96fc34991f68266805c14ec0f0e321c01c05c58/automations/js/prettier.config.js) for some reason.

**Comment by obulat** [comment]
> @obulat automations have [their separate Prettier config file](https://github.com/WordPress/openverse/blob/b96fc34991f68266805c14ec0f0e321c01c05c58/automations/js/prettier.config.js) for some reason.

Great find! I guess we need to update the pre-commit to include `mjs`, and remove this prettier config.

**Pull request merged**